recipes/dd-handler.rb

```diff
@@ -46,11 +46,20 @@
   ENV['DATADOG_PROXY'] = proxy_url.to_s
 end
 
+extra_endpoints = []
+node['datadog']['extra_endpoints'].each do |_, endpoint|
+  next unless endpoint['enabled']
+  endpoint.delete('enabled')
```

Comment:
about my comment on the `Mash` thing, not sure it's super important since in the handler we copy the config to a `Mash` since v0.5.0 (see https://github.com/DataDog/chef-handler-datadog/pull/50)

I'd be surprised if anyone still uses `chef-handler-datadog` < `0.5.0`...